### ISSUE TITLE
DPSR error in training

### ISSUE THREAD

**Issue by pjh1023** [cszn/KAIR#3]
/home/pjh/anaconda3/envs/DPSR/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:100: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)

I am keep getting this error when I'm trying to run the training code (main_train_dpsr.py).
I've seen the issue#2 that deals with the similar error but I'm wondering if the problem is also solved in the DPSR code! Do you have any solutions?

**Comment by cszn** [comment]
It is UserWarning. You can ignore it.

**Comment by pjh1023** [comment]
Thanks for your quick answer!
